src/components/user/PersonalProfileEditForm.js

```diff
@@ -9,9 +9,29 @@ import {
   TextField,
   Button,
 } from '@mui/material';
-import { Avatar, Alert } from '@mui/material';
+import { Alert } from '@mui/material';
 import axiosInstance from '../../apis/axiosClient';
 import { useNavigate } from 'react-router-dom';
+
+const ProfileImage = ({ user }) => {
+  if (user && user.profileImage && user.profileImage.url) {
+    return (
+      <img
+        src={user.profileImage.url}
+        alt={user.profileImage.altText || 'user profile image'}
+        className="img-fluid rounded-circle"
+        style={{
+          width: '100px',
+          height: '100px',
+          maxWidth: '100px',
+          maxHeight: '100px',
+        }}
+      />
+    );
+  }
+  return null;
+};
+
```

Comment:
Rather than using what seems to be the same code in two separate files; it would be best to export the `ProfileImage` component from the UserProfile file (or write it in its own file to be imported in both UserProfile and PersonalProfileeditForm) this allows us to reuse the same code and ensure that the visuals are consistent across components.